### PR TITLE
Add more faces

### PR DESCRIPTION
**Pull request by elken**
Add a couple more faces needed for HTML export in ox-chameleon

Marked as draft until copyright assignment is finalised

### CODE REVIEW

**Comment by elken** [comment]
Copyright assignment resolved, if required teco I can send you my documents privately.

**Comment by tecosaur** [comment]
Great!

File: engrave-faces.el [hunk 1]
```diff
@@ -287,6 +291,17 @@ This function is lifted from htmlize."
        (font-lock-regexp-grouping-backslash :short "fl-regexp-backslash" :slug "rb"                        :weight bold)
        ;; org-faces.el
        (org-block                           :short "org-block"           :slug "ob") ; forcing no background is preferable
+       (org-block-begin-line                :short "org-block-begin"     :slug "obb") ; forcing no background is preferable
+       (org-block-end-line                  :short "org-block-end"       :slug "obe") ; forcing no background is preferable
+       ;; outlines
+       (outline-1 :foreground blue                        :short "o-1")
```
Comment (tecosaur) [inline engrave-faces.el:297]: I'm not planning on adding a doom colour dependancy.

File: engrave-faces.el [hunk 2]
```diff
@@ -287,6 +291,17 @@ This function is lifted from htmlize."
...
+       (outline-2 :foreground magenta                     :short "o-2")
+       (outline-3 :foreground violet                      :short "o-3")
+       (outline-4 :foreground (doom-lighten blue 0.25)    :short "o-4")
+       (outline-5 :foreground (doom-lighten magenta 0.25) :short "o-5")
+       (outline-6 :foreground (doom-lighten blue 0.5)     :short "o-6")
+       (outline-7 :foreground (doom-lighten magenta 0.5)  :short "o-7")
+       (outline-8 :foreground (doom-lighten blue 0.8)     :short "o-8")
```
Comment (tecosaur) [inline engrave-faces.el:304]: I'm not planning on adding a doom colour dependancy.

Comment (tecosaur) [reply engrave-faces.el:304]: I'd also like it if we had outline/slug like `outline-1`/`oa`.

Comment (elken) [reply engrave-faces.el:304]: > I'm not planning on adding a doom colour dependancy.

That's on me, I meant to evaluate it to get the proper colour code before I pushed

> I'd also like it if we had outline/slug like outline-1/oa.

I did add them but it seems they were lost in another commit, oops

Comment (tecosaur) [reply engrave-faces.el:304]: > That's on me, I meant to evaluate it to get the proper colour code before I pushed

Note that the default style is supposed to be the values using the `default` theme.

Comment (elken) [reply engrave-faces.el:304]: > > That's on me, I meant to evaluate it to get the proper colour code before I pushed
> 
> Note that the default style is supposed to be the values using the `default` theme.

I know, I've been evaluating in vanilla emacs for the most part

**Comment by tecosaur** [comment]
The outline slugs can't contain numbers, that's why I recommended `oa` (`ob`, `oc`, ...).
I'd also like the outline shorts to `outline-N` instead of `o-N`.
It would also be good if the alignment matched.

Lastly, I'm wondering if `:family` support should be added to LaTeX with this? Perhaps later.

**Comment by elken** [comment]
> The outline slugs can't contain numbers, that's why I recommended `oa` (`ob`, `oc`, ...). I'd also like the outline shorts to `outline-N` instead of `o-N`. It would also be good if the alignment matched.

Oh I misunderstood there, I get what you mean now.

> Lastly, I'm wondering if `:family` support should be added to LaTeX with this? Perhaps later.

Absolutely, most of what I've added here should be in LaTeX if it's not already. There's still more that _can_ be done here, eg better prettify/org-modern replacement and whatever other packages could apply styles.

**Comment by tecosaur** [comment]
Why did me force pushing auto-close this?

**Comment by tecosaur** [comment]
Oh, I pushed the wrong commit, and now I get permission denied when I re-push :(

**Comment by elken** [comment]
_uh oh_

**Comment by tecosaur** [comment]
Pushed in 94fbdb021f0504982d6902b8df2b0a5c79a1d9da

**Pull request closed without merging**